packages/core/src/odata-common/request-builder/batch/batch-response-deserializer.spec.ts

```diff
@@ -43,37 +43,39 @@ describe('batch response transformer', () => {
     const entityToConstructorMap = {
       entity: 'entity' as any
     };
-    const batchTransformer = new BatchResponseDeserializer(
-      entityToConstructorMap,
-      responseDataAccessorV2,
-      {} as EntityDeserializer
-    );
+    describe('v2', () => {
```

Comment:
I'm not a big fan of nested `describe`s. This caused errors in the past (but it may have been with jasmine not jest...) Maybe it would be better to start a new block here to flatten it a bit rather than nesting deeper